functional-tests/base-test.py

```diff
@@ -0,0 +1,29 @@
+import os
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
+
+# Setting up saucelabs config
+username = os.environ["SAUCE_USERNAME"]
+access_key = os.environ["SAUCE_ACCESS_KEY"]
+capabilities = {
+    'platform': "Mac OS X 10.9",
+    'browserName': "chrome",
+    'version': "31",
+}
+capabilities["tunnel-identifier"] = os.environ["TRAVIS_JOB_NUMBER"]
+hub_url = "%s:%s@localhost:4445" % (username, access_key)
+driver = webdriver.Remote(desired_capabilities=capabilities, command_executor="https://%s/wd/hub" % hub_url)
+
+
+# Tests will be run from here
```

Comment:
This will be our entry point to start running all of our tests, we can/should break this up into different modules.